services/wallet/common/log_parser.go

```diff
@@ -537,32 +616,27 @@ func ExtractTokenIdentity(dbEntryType Type, log *types.Log, tx *types.Transactio
 	}
 
 	switch correctType {
-	case EthTransfer:
-		if tx != nil {
-			txValue = new(big.Int).Set(tx.Value())
-		}
 	case Erc20Transfer:
 		tokenAddress = new(common.Address)
 		*tokenAddress = log.Address
-		from, to, value := ParseErc20TransferLog(log)
-		txValue = value
+		from, to, _ := ParseErc20TransferLog(log)
 		txFrom = &from
 		txTo = &to
 	case Erc721Transfer:
 		tokenAddress = new(common.Address)
 		*tokenAddress = log.Address
-		from, to, tokenID := ParseErc721TransferLog(log)
-		txTokenID = tokenID
+		from, to, _ := ParseErc721TransferLog(log)
 		txFrom = &from
 		txTo = &to
-	case Erc1155SingleTransfer:
+	case Erc1155Transfer:
 		tokenAddress = new(common.Address)
 		*tokenAddress = log.Address
-		_, from, to, tokenID, value := ParseErc1155TransferSingleLog(log)
-		txTokenID = tokenID
+		_, from, to, _, _, err := ParseErc1155TransferLog(log, Erc1155TransferSingleEventType)
```

Comment:
Hmm I understand it doesn't matter since `from` and `to` extraction is the same for the batch and single transfers, but hardcoding type `Erc1155TransferSingleEventType` here is a bit confusing. You could add a comment about that